src/CommunityToolkit.Maui/ReadMe.txt

```diff
@@ -0,0 +1,31 @@
+﻿.NET MAUI Community Toolkit
+
+## Initializing
+
+In order to use the .NET MAUI Community Toolkit you need to call the extension method in your `MauiProgram.cs` file as follows:
+
+```csharp
+public static MauiApp CreateMauiApp()
+{
+    var builder = MauiApp.CreateBuilder();
+
+    // Initialise the toolkit
+	builder.UseMauiApp<App>().UseMauiCommunityToolkit();
+
+    // the rest of your logic...
+}
+```
+
+## XAML usage
+
+In order to make use of the toolkit within XAML you can use this namespace:
+
+xmlns:mct="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
```

Comment:
Let's recommend using `toolkit` instead of `mct`.

Devs can use whichever keyword they prefer, but, long story short, we'd get in trouble with marketing/legal for recommending `mct`.

```suggestion
xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
```

Reply:
I'm happy to suggest `toolkit` instead. What do you think to our switch to using the `mct` which will default to `xmlns:mct=...` here?https://github.com/CommunityToolkit/Maui/blob/7bfe894f5a9f22c5cc1293ed9d0220667adf24e6/src/CommunityToolkit.Maui/AssemblyInfo.shared.cs#L5

Reply:
Good catch! Yea, let's update it to `toolkit` in this PR 👍